### ISSUE TITLE
No documentation on how to connect using aad in management API

### ISSUE THREAD

**Issue by ilya-git** [Azure/azure-signalr#1433]
### Describe the bug
[This](https://github.com/Azure/azure-signalr/releases/tag/v1.10.0) migration guide mentions a new constructor for `ServiceEndproint` that allows for add connection (I assume `DefaultAzureCredential`), but there is no documentation that I could find that mentions on how to use that with management api. 

Maybe it's not implemented at all and that's why I cannot find it? It would be nice to have docs on this. I would appreciate as well if you can post in here how to that before updating the docs :)

### Further technical details
- Your Azure SignalR SDK version : 1.10.0

P.S. I think the `ServiceEndpoints` option can be used, but I did not find any docs on it.

**Comment by Y-Sindo** [comment]
Yes you are right, `ServiceEndpoints` option can be used, but 1.10.0 management SDK still has an issue with transient transport type (the default transport type) preventing from using `ServiceEndpoints`. The issue will be fixed in 1.11.0 version which will be released in this week. We'll also provide detailed document in the future.

If you want to try the `DefaultAzureCredential` now, you can switch to `Persistent` transport type and use `ServiceEndpoints` option. Here is the sample code:
```cs
    var serviceManagers = new ServiceManagerBuilder()
    .WithOptions(o =>
    {
        o.ServiceEndpoints = new ServiceEndpoint[] { new ServiceEndpoint(new Uri("https://<your-signalr-service-name>.service.signalr.net"), new DefaultAzureCredential()) };
        o.ServiceTransportType = ServiceTransportType.Persistent;
    })
    .BuildServiceManager();
```

**Comment by ilya-git** [comment]
Thanks a lot! This approach does work and luckily enough I was already using persistent transport type, as I suppose it is more efficient.
